### PR TITLE
add postman.md, inserting with using postman

### PR DESCRIPTION
**Pull request by kimsama**
Add a page describing sending sample xAPI statement of the Getting Started page with Postman.

### CODE REVIEW

**Comment by cliffcaseyyet** [comment]
Thank you for adding a postman page, this could be useful to implementers. Can you add links to this either in the Getting Started or in the main README section under `Reference`? Also I have a few typo comments which I will post now.

Thanks,
Cliff

File: doc/postman.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Postman 
+
+This page describes how to send an xAPI statement through [Postman](https://www.postman.com/), a well-known API platform for building and using APIs.
+
+First, install and run Postman. You may need to sign up if that is the firs time running of Postman.
```
Comment (cliffcaseyyet) [inline doc/postman.md:5]: "if this if your first time running Postman."

File: doc/postman.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Username: my_key (or created key in the **Credential Management** page in the yetXApiSQLRS)
+* Password: my_secret (or create secret in the **Credential Management** page in the yetXApiSQLRS)
+
+![postman autho](images/postman_auth.png)
+
+#### 4. Set Body
+
+Go to the "Body" section, select the "raw" option, and choose the "JSON (application/json)" content type.
+
+Copy and paste your JSON data into the body:
```
Comment (cliffcaseyyet) [inline doc/postman.md:40]: "Copy and paste your valid JSON xAPI Statement(s) into the body:"

Comment (kimsama) [reply doc/postman.md]: Applied

**Comment by kimsama** [comment]
I updated. The links are added on both README.md and startup.md.

**Review by cliffcaseyyet — APPROVED** [review]
Awesome, thanks for contributing!

**Pull request merged**